src/backend/marsha/urls.py

```diff
@@ -52,7 +54,7 @@
     ),
     path("api/", include(router.urls)),
     path("xapi/", XAPIStatementView.as_view(), name="xapi"),
-    path("", SiteView.as_view(), name="site"),
+    re_path("site/.*", SiteView.as_view(), name="site"),
```

Comment:
Why changing this url ?

Reply:
This is a change for local development, my mistake to put it here.

There's an issue where you need to be able to refresh the page when you're in the frontend site and Django needs to redirect to the site view no matter the frontend URL used, but it should not be part of this PR :)